simSPI/linear_simulator/shift_utils.py

```diff
@@ -0,0 +1,90 @@
+"""Module containing method to shift an image in Fourier domain."""
+import numpy as np
+import torch
+
+
+class Shift(torch.nn.Module):
+    """class containing method to shift an image in Fourier domain.
```

Comment:
NIT: class containing a method to apply a spatial shift on an image in Fourier domain.